### PR TITLE
remove meta and format html

### PR DESCRIPTION
**Pull request by tumpBlon**

### CODE REVIEW

**Comment by tumpBlon** [comment]
It's for test!
### Test
**Test**
_Test_

> test

> test
`const test = () = console.log('test')`
[https://github.com/Emil-Mirzoev/Lesson/pull/1](url)

- 1
- 2
- 3

1. 1
2. 2
3. 3

- [ ] test
- [ ] 
@Emil-Mirzoev hi
#1

File: index.html [hunk 1]
```diff
@@ -1,114 +1,163 @@
...
-        </nav>
-    <div class="logo"><a href="#"></a></div>
-        <nav class="header_nav">
-            <ul class="header_menu">
-                <li class="header_menu-item"><a href="">Blog</a></li>
-                <li class="header_menu-item"><a href="">Integration</a></li>
-                <li class="header_menu-item"><a href="">Contact</a></li>
-            </ul>
-        </nav>
+      <nav class="header_nav">
```
Comment (tumpBlon) [inline index.html:12]: test

File: index.html [hunk 2]
```diff
@@ -1,114 +1,163 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (tumpBlon) [inline index.html:6]: test

File: index.html [hunk 3]
```diff
@@ -1,114 +1,163 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="normalize.css">
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
+    <link rel="stylesheet" href="normalize.css" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
```
Comment (Emil-Mirzoev) [inline index.html:10]: thx

**Pull request merged**